tests/unit/json/org2_example

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
The original was referenced from test_events.py. The new one contains different data and is referenced from test_orgs.py.